README.md

```diff
@@ -26,7 +26,7 @@ Use ``--help`` to learn more usage details.
 
 First of all, this is a FlexMeasures plugin. Consult the FlexMeasures documentation for setup.
 
-1. Add the path to this directory to your FlexMeasures (>v0.4.0) config file,
+1. Add `/path/to/flexmeasures-entsoe/flexmeasures_entsoe` to your FlexMeasures (>v0.4.0) config file,
```

Comment:
... "or just `flexmeasures_entsoe` if you installed this plugin as a package (see below).

Reply:
this is correct.